.github/workflows/backup-live-image.yml

```diff
@@ -10,36 +10,22 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
       - name: Get the live release tag
         id: get_tag
         run: echo ::set-output name=BRANCH_TAG::$(echo $GITHUB_REF | cut -d / -f 3)
 
-      - name: Backup live release
-        env:
-          COMMIT_SHA: ${{ github.sha }}
-          DOCKER_REGISTRY_URL: ntbscontainerregistry.azurecr.io
-          DOCKER_REGISTRY_AZURE_NAME: ntbsContainerRegistry
-          DOCKER_REPOSITORY: ntbs-service
-          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
-          BACKUP_DOCKER_REGISTRY_URL: ntbsbackups.azurecr.io
-          BACKUP_DOCKER_REPOSITORY: ntbs-backups
-          BACKUP_DOCKER_USERNAME: ${{ secrets.BACKUP_DOCKER_USERNAME }}
-          BACKUP_DOCKER_PASSWORD: ${{ secrets.BACKUP_DOCKER_PASSWORD }}
-          BACKUP_IMAGE_TAG: ${{ steps.get_tag.outputs.BRANCH_TAG }}
-        run: |
-          image_tag=$(az acr repository show-tags -n $DOCKER_REGISTRY_AZURE_NAME --repository $DOCKER_REPOSITORY -u $DOCKER_USERNAME -p $DOCKER_PASSWORD | grep $COMMIT_SHA | cut -d \" -f 2)
```

Comment:
I couldn't find a way to list the tags from the github registry, so I had to change the way that this workflow works - instead of pulling the image to backup from the container registry, it now re-builds the image from the source code.